docs/_importers/pluxml.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: PluXML
+prev_section: mt
+link_source:  pluxml
+next_section: posterous
+permalink: /docs/pluxml/
+---
+
+To import your posts and drafts from a PluXML blog, run:
+
+{% highlight bash %}
+$ ruby -r rubygems -e 'require "jekyll-import";
+    JekyllImport::Importers::Pluxml.run({
+      "source" => "/pluxml/data/articles",
+      "layout" => "your_layout",
+      "avoid_liquid" => TRUE
```

Comment:
Using `true` here instead of `TRUE` is recommended, just FYI.

```irb
~$ irb
TRUE
>> TRUE
(irb):1: warning: constant ::TRUE is deprecated
=> true
```

Reply:
@parkr Right, I've changed it to lowercase.